src/common/CMakeLists.txt

```diff
@@ -13,7 +13,11 @@ endif()
 if(OQS_USE_AES_OPENSSL)
     set(AES_IMPL aes/aes_ossl.c)
 else()
-    set(AES_IMPL aes/aes_c.c)
+   set(AES_IMPL aes/aes.c aes/aes_c.c)
+    if (OQS_USE_CPU_EXTENSIONS)
+      set(AES_IMPL ${AES_IMPL} aes/aes_ni.c)
+      add_compile_options(-maes)
```

Comment:
Can you change this to `set_source_files_properties(aes/aes_ni.c PROPERTIES COMPILE_FLAGS -maes)`?